Sources/SpeziAccessGuard/AccessGuardView.swift

```diff
@@ -16,20 +16,19 @@ struct AccessGuardView<GuardedView: View>: View {
 
     
     var body: some View {
-        guardedView
-            .overlay {
-                if viewModel.locked {
-                    EnterCodeView(viewModel: viewModel)
-                        .ignoresSafeArea(.container)
-                }
-            }
-            .onAppear {
-                if viewModel.locked && viewModel.configuration.guardType == .biometrics {
-                    Task {
-                        try? await viewModel.authenticateWithBiometrics()
+        if viewModel.locked {
```

Comment:
Only thing I want to investigate: Does this have any impact on the layout of the view (e.g. it doesn't fill the screen by default or some other implications)?

Would be great if we can add some regression tests in the testing app & assess that in the future, e.g., for a few that only has a smaller size or something similar. Would be great to explore that impact before we tag a new release so it doesn't break anything in the apps that use it.

Maybe adding some snapshot tests like the ones in SpezViews would be the best thing we could do here?

Reply:
I don't think this is a problem. The `guardedView` and `EnterCodeView` are shown within the context of AccessGuard, so whatever bounds it has will be reflected in these views. Did I understand your concern correctly?

Nevertheless, I added the infrastructure in place and some snapshot tests. I tried snapshot testing AccessGuard as a sheet, but `swift-snapshot-testing` has a bug that does not capture the content of the sheet. This [issue](https://github.com/pointfreeco/swift-snapshot-testing/issues/279) is still unresolved.